src/server.cpp

```diff
@@ -258,6 +259,16 @@ int server_main(int argc, char** argv) {
         auto& config = ovms::Config::instance().parse(argc, argv);
         configure_logger(config.logLevel(), config.logPath());
 
+#ifdef MTR_ENABLED
+        const char* traceFileName = config.tracePath().c_str();
+#else
+        const char* traceFileName = "";
+#endif
+        if (!profiler_init(traceFileName)) {
+            SPDLOG_ERROR("Cannot open file for profiler, --trace_path: {}", traceFileName);
+            return EXIT_FAILURE;
+        }
+
```

Comment:
Shouldn't it be:
```suggestion
#ifdef MTR_ENABLED
        const char* traceFileName = config.tracePath().c_str();
        const char* traceFileName = "";
        if (!profiler_init(traceFileName)) {
            SPDLOG_ERROR("Cannot open file for profiler, --trace_path: {}", traceFileName);
            return EXIT_FAILURE;
        }
        #endif
```
Anyway maybe instead of making ifdef for catch clauses use guard structure?

Reply:
Added guard